Sources/Sentry/SentryTracer.m

```diff
@@ -93,6 +96,10 @@ - (instancetype)initWithTransactionContext:(SentryTransactionContext *)transacti
             initFrozenFrames = currentFrames.frozen;
         }
 #endif
+        if ([_hub getClient].options.enableProfiling) {
```

Comment:
Is this the correct way to check options? `SentryOptions` aren't directly passed down to the tracer.

Reply:
I would add a reference to `SentrySDK+Private.h`  instead of `SentryClient.h` and then use SentrySDK.options. But at the end is the same object.